examples/visual-tests/amp-story/amp-story-page-attachment.html

```diff
@@ -49,23 +49,114 @@
         poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
         poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg"
         supports-landscape>
-      <amp-story-page id="cover">
+      <amp-story-page id="inline-default">
         <amp-story-page-attachment layout="nodisplay">
           <h1>Some really cool video</h1>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
           </p>
-          <amp-video width="auto" height="270" autoplay src="/examples/visual-tests/video/tokyo.mp4" controls>
-          </amp-video>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="inline-custom-text">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          cta-text="Custom text">
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="inline-dark-theme">
+        <amp-story-page-attachment
+          layout="nodisplay"
+          theme="dark">
+          <h1>Some really cool video</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          </p>
+        </amp-story-page-attachment>
+      </amp-story-page>
+
+      <amp-story-page id="inline-dark-theme-1-image">
+        <amp-story-page-attachment
+          title="Lorem ipsum"
+          layout="nodisplay"
+          cta-image="noimage"
```

Comment:
It looks like we can link to our images like this:

```suggestion
          cta-image="../../amp-story/img/sweater.jpg"
```

I found a reference in a different visual test in the same folder:
https://github.com/ampproject/amphtml/blob/main/examples/visual-tests/amp-story/amp-story-consent.html#L79